frontend/src/main/components/SectionsTableBase.js

```diff
@@ -30,7 +30,7 @@ export default function SectionsTableBase({ columns, data, testid = "testid"}) {
           prepareRow(row)
           return (
             <>
-            {row.cells[0].isGrouped || (!row.cells[0].isGrouped && row.allCells[3].value) ? 
+            {row.cells[0].isGrouped || (!row.cells[0].isGrouped && row.allCells[4].value) ? 
```

Comment:
I'm confused about this magic number [4] here.  Can you explain what's going on?

Reply:
If 4 is the position in the row of the cell containing the Section number, for example, it might make the code less confusing if we did something like:

```js
const SECTION_NUMBER_COLUMN = 4;
```

And then this was: 

```
{row.cells[0].isGrouped || (!row.cells[0].isGrouped && row.allCells[SECTION_NUMBER_COLUMN].value) ?
```

But I'm just speculating about what the `4` means.   Can you help?

Reply:
We should try to explain the `0` too in the same way with a good choice of variable name.

Reply:
Previously it was 
`{row.cells[0].isGrouped || (!row.cells[0].isGrouped && row.allCells[3].value) ? ` where 0 denotes the column of `COURSE_ID` and 3 denotes `ENROLLED_NUM`. So when we add another column for section number, we need to modify the index of the column of `Enrolled` from 3 to 4.